### DISCUSSION TITLE
Example of report is not available - link broken

### DISCUSSION THREAD

**Discussion by mariaklimenko** [allure-framework/allure2#1875] [TestOps]
Hello! I am investigating how to show load test results in Allure TestOps and found this plugin: https://github.com/biski/gatling2allure

In the description, there is a link to the sample output but the link is broken and the content doesn't exist anymore: https://ci.qameta.in/job/allure2/job/master/Demo2_Report/

Can you please fix the link or maybe point me to another place if it was relocated?  Thanks!

**Comment by cheshi-mantu** [comment]
Hi @mariaklimenko 
This repo is not an official solution from Allure Framework's project. Currently, there is no officially supported integration to import load testing tools' results to Allure TestOps. But stay tuned. This feature is planned.
